core/src/main/java/org/apache/carbondata/core/scan/scanner/AbstractBlockletScanner.java

```diff
@@ -46,6 +48,10 @@
 
   private AbstractScannedResult emptyResult;
 
+  private static int NUMBER_OF_ROWS_PER_PAGE = Integer.parseInt(CarbonProperties.getInstance()
```

Comment:
if declare as `static`, it can be changed dynamically after system is initialized